DependencyInjection/DoctrineMongoDBExtension.php

```diff
@@ -496,4 +507,11 @@ public function getXsdValidationBasePath()
     {
         return __DIR__ . '/../Resources/config/schema';
     }
+
+    private function buildDeprecationArgs(string $version, string $message): array
+    {
+        return method_exists(BaseNode::class, 'getDeprecation')
```

Comment:
Please leave a `@todo` noting what's the minimal Symfony version we can drop the if, that'll be handy in the future

Reply:
Added note about minimal Symfony version supporting this (5.1).